client/gutenberg/extensions/map-block/component.js

```diff
@@ -0,0 +1,343 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { __ } from '@wordpress/i18n';
+import { Component, createRef, Fragment } from '@wordpress/element';
+import { Button, Dashicon, TextareaControl, TextControl } from '@wordpress/components';
+import { get, assign } from 'lodash';
```

Comment:
Sadly this format of importing `lodash` is currently causing fatal errors in Gutenberg, since it causes us to clash with `underscores` library used by wp-admin Media components.

We should use this format for now instead:
```js
import get from 'lodash/assign';
import get from 'lodash/get';
```

See https://github.com/Automattic/wp-calypso/pull/28157 for another example.

I hope we can change this back to this format soonish.

Reply:
Yeah, saw that, will change. Does this apply to `lodash` imports exclusively?

Reply:
Resolved in https://github.com/Automattic/wp-calypso/pull/27836/commits/ee85e9bab8a71a126b62cfb28705df545f46e730 and https://github.com/Automattic/wp-calypso/pull/27836/commits/d2c809e3f45eaf3a76f33ef56df0e6812a46323e. This was why even though the Map block functioned well other core blocks failed (e.g. Image). Initially assumed this was unrelated to this branch, but turns out the `lodash` imports were the culprit. Thank you @simison for pointing this out!